.gitignore

```diff
@@ -1,13 +1,13 @@
-/coverage
-/demo/dist
+coverage/
+demo/dist/
 
 # Generated build files
-/es
-/lib
-/umd
+es/
+lib/
+umd/
 
 # Node
-/node_modules
+node_modules/
```

Comment:
Huh, out of curiousity, why the change of slash location?

Reply:
hmm I was having some issues and it was picking up `node_modules`